packages/components/pie-radio/src/index.ts

```diff
@@ -12,9 +15,50 @@ const componentSelector = 'pie-radio';
 /**
  * @tagname pie-radio
  */
-export class PieRadio extends RtlMixin(LitElement) implements RadioProps {
+export class PieRadio extends FormControlMixin(RtlMixin(LitElement)) implements RadioProps {
+    @property({ type: Boolean })
+    public checked = defaultProps.checked;
+
+    @property({ type: Boolean })
+    public defaultChecked = defaultProps.defaultChecked;
+
+    @property({ type: Boolean })
+    public disabled = defaultProps.disabled;
+
+    @property({ type: String })
+    public name: RadioProps['name'];
```

Comment:
```suggestion
    public name?: RadioProps['name'];
```

Reply:
This is not required in the index file because the `RadioProps` type already specifies the type as optional.

![2024-09-17 08_59_29](https://github.com/user-attachments/assets/4a55f84e-9bed-4a5a-b3c9-005d78cec6f8)